src/xlsx.js

```diff
@@ -0,0 +1,20 @@
+let XLSX; // set lazily
+
+export default async function xslx(require) {
+  return XLSX = await require("xlsx@0.17.0/dist/xlsx.mini.min.js");
+}
+
+export class XlsxWorkbook {
+  constructor(workbook) {
+    Object.defineProperties(this, {
+      _: {value: workbook}
```

Comment:
It'd be nice to expose the XLSX or XLSX.utils somewhere so people can easily use those functions like sheet_to_html or sheet_to_formulae. Or for modifying and writing back out to xlsx.

Reply:
Right, I commented on why I didn’t do this: there are two versions of XLSX (mini and full), and we only want the mini one to parse XLSX files, but I don’t think that one should be the one in standard library. I suppose I could hide it in the XlsxWorkbook class I made, but I figured it would be better to expose a minimal API that is easier for us to improve over time. (In my opinion, the design of SheetJs is not a little weird… I’m not sure why it favors global static functions over methods.)

Reply:
ok, let's start conservatively and see if we need to add more. there's also always the workaround of notebook-space requiring xlsx/sheetjs explicitly and using it if you need to.